.github/workflows/tests.yml

```diff
@@ -37,7 +37,28 @@ jobs:
     - name: Build Portable Ruby
       run: |
         if [ "$RUNNER_OS" = "Linux" ]; then
-          docker run -v"$PWD":/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-portable-ruby homebrew-portable-x86_64 brew portable-package portable-ruby
+          docker run --name=homebrew-portable-ruby -w /bottle -v"$PWD":/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-portable-ruby homebrew-portable-x86_64 brew portable-package portable-ruby
+          docker cp homebrew-portable-ruby:/bottle .
+          mv bottle/* .
+          rmdir bottle
         else
           brew portable-package portable-ruby
         fi
+
+    - name: Install Portable Ruby
+      run: |
+        version=2.6.3
+        if [ "$RUNNER_OS" = "Linux" ]; then
+          prefix=/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor
+        else
+          prefix=/usr/local/Homebrew/Library/Homebrew/vendor
+        fi
+        mkdir -p $prefix
+        tar -C $prefix -xf portable-ruby--*.tar.gz
+        ln -sf $version $prefix/portable-ruby/current
+        echo $version >$prefix/portable-ruby-version
+
+    - name: Test Portable Ruby
+      run: |
+        brew config
+        brew tests
```

Comment:
The main issue I see with the `brew tests` here is: what do you do if it fails? In the case of https://github.com/Homebrew/brew/pull/6556 there were some tests (extracted to https://github.com/Homebrew/brew/pull/6558) that could be fixed before 2.6.3 was merged and worked on 2.6.3 and 2.3.7. Unfortunately, though, there were some tests (https://github.com/Homebrew/brew/pull/6556/commits/e3ee64a5fd59f4f3203b369424bf8347b563cb11) which couldn't be fixed to work consistently on both versions. That would mean that neither this repo nor Homebrew/brew can have green CI until both have been updated at once (or the bottle is built from a WIP branch here, shipped to `master` on Homebrew/brew and then the CI rerun and hope it passes).

Reply:
In this case, you could temporarily change brew clone url to a forked in CI here. So you can test both portable-ruby and brew at the same time.